src/pages/Tasks/Tasks.js

```diff
@@ -1,31 +1,102 @@
-import React from "react";
+import React, { useState } from "react";
+import { v4 as uuidv4 } from "uuid";
 import "./task.css";
-import tasksDB from "../../tasksDB";
+import { Link } from "react-router-dom";
```

Comment:
try to follow the imports rule, wherein put all the library imports at the top and custom imports after that